app/controllers/order_details_controller.rb

```diff
@@ -120,4 +121,15 @@ def order_detail_params
     params.require(:order_detail).permit(:account_id)
   end
 
+  def redirect_to_path
```

Comment:
If we use this same logic in 3 or more places, I'd vote for putting this into a concern